vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -109,6 +113,20 @@ def __init__(
         # mapping of request ID to sampling params
         self._sampling_params_by_request: dict[str, SamplingParams] = {}
         self._max_logprobs: Optional[int] = None
+        
+        self.requests: dict[str, CachedRequestState] = {}
```

Comment:
These should be unused now, right?
```
        self._sampling_params_by_request: dict[str, SamplingParams] = {}
        self._max_logprobs: Optional[int] = None`
```

Also these three attributes are holding state information about the batch:
```
        # position_ids of all the sequences in current batch
        self._position_ids: torch.Tensor = None
        # attention masks of all the sequences in current batch
        self._mask: torch.Tensor = None
        # mapping: request id to index in batch
        self._req_ids2idx: dict = {}
```
It would be great to get that data from `InputBatch` so that we don't have to track state in multiple places